pkg/executables/cmk.go

```diff
@@ -189,35 +189,37 @@ func (c *Cmk) ValidateZonesPresent(ctx context.Context, zones []v1alpha1.CloudSt
 	return zoneIdentifiers, nil
 }
 
-func (c *Cmk) ValidateDomainPresent(ctx context.Context, domain string) (v1alpha1.CloudStackResourceIdentifier, error) {
```

Comment:
These changes seem to be clubbing with https://github.com/aws/eks-anywhere/pull/1750#issuecomment-1087892888

What's the plan?